packages/react-instantsearch-hooks/src/index.ts

```diff
@@ -35,4 +35,3 @@ export * from './connectors/useStats';
 export * from './connectors/useToggleRefinement';
 export * from './hooks/useConnector';
 export * from './hooks/useInstantSearch';
-export * from './lib/warn';
```

Comment:
other usages of warn need to be removed too

Reply:
Ah, I didn't notice it was used by other code sections after I exported it. Reintroducing it so it doesn't break the build, we'll move it later.